bundles/org.openhab.binding.doorbird/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -4,29 +4,27 @@
 	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
 
 	<config-description uri="thing-type:doorbird:config">
-		<parameter name="doorbirdId" type="text" required="true">
-			<label>ID</label>
-			<description>Doorbird Identifier (e.g. 1)</description>
-		</parameter>
 		<parameter name="doorbirdHost" type="text" required="true">
 			<label>Host</label>
 			<description>Hostname or IP address of Doorbird</description>
 			<context>network-address</context>
 		</parameter>
 		<parameter name="userId" type="text" required="true">
 			<label>User ID</label>
+			<description>Doorbird user ID with API permissions enabled</description>
 		</parameter>
 		<parameter name="userPassword" type="text" required="true">
 			<label>Password</label>
+			<description>Doorbird user password</description>
 			<context>password</context>
 		</parameter>
 		<parameter name="imageRefreshRate" type="integer" min="2" max="600">
 			<label>Image Refresh Rate</label>
 			<description>Image refresh rate in seconds (blank to disable)</description>
 		</parameter>
 		<parameter name="doorbellOffDelay" type="integer" min="1">
-			<label>Doorbell Off Delay</label>
-			<description>Delay to set doorbell channel OFF after doorbell event (blank to disable)</description>
+			<label>Doorbell Released Delay</label>
+			<description>Delay after a doorbell event to send RELEASED event (blank to disable)</description>
```

Comment:
I would mention what kind of units (seconds, minutes, etc...) this refers to.

Reply:
Good point. I think there might be a couple places where the units should be specified. I'll make sure to change them all.